test/BUILD

```diff
@@ -23,6 +23,7 @@ java_library(
     deps = [
```

Comment:
As mentioned previously, the CI for this repo is insufficient to verify correctness. Have we tested these changes? If so, how/where?

In general, I would say it's worth including this information in the PR description and/or comments.

Reply:
We have! All of typedb-client-java's tests run successfully with this version of typedb-common. Would it make sense to just say that this is the case or to also post the test output?

Reply:
No need - that last comment "All of typedb-client-java's tests run successfully with this version of typedb-common" is enough by itself!